Classes/Repository/RepositoryIssuesViewController.swift

```diff
@@ -81,20 +81,40 @@ IndicatorInfoProvider {
             nextPage: page as String?,
             containerWidth: view.safeContentWidth(with: feed.collectionView)
         ) { [weak self] (result: Result<RepositoryClient.RepositoryPayload>) in
+            guard let `self` = self else { return }
             switch result {
             case .error:
-                self?.error(animated: trueUnlessReduceMotionEnabled)
+                self.error(animated: trueUnlessReduceMotionEnabled)
             case .success(let payload):
+                if let itemCount = payload.numberOfItems,
+                      itemCount > 0 {
+                    self.title = self.composeIndicatorTitle(with: itemCount)
```

Comment:
Hm, seeing this now which confuses me — this updates the view controller's title, not the pager tab, does it? I can't seem to build the app at the moment, so can't check it myself.

Reply:
Hmm, yes the indicator is using the viewController title, I used the same pattern used before check the last method within this class. Or maybe I misunderstood your question.

Reply:
I'm currently using Xcode 10.2 and the app builds normally, am I supposed to use a specific Xcode version?